{"name": "Verify API handles invalid endpoint", "status": "passed", "steps": [{"name": "Given the API client is initialized for TC_01 Third Scenario", "status": "passed", "start": 1739648771456, "stop": 1739648771457}, {"name": "When I send a GET request to an invalid endpoint", "status": "passed", "start": 1739648771457, "stop": 1739648772081}, {"name": "Then the response status should be 404", "status": "passed", "start": 1739648772081, "stop": 1739648772081}], "start": 1739648771455, "stop": 1739648772082, "uuid": "f464ae72-5257-4d35-a04b-fa7a3f79bcd8", "historyId": "ee47dd7b8c3888cdf3c79429f5c48e96", "fullName": "GET User List, User Login and Invalid Endpoint Test: Verify API handles invalid endpoint", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "GET User List, User Login and Invalid Endpoint Test"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}